### DISCUSSION TITLE
How to install a more recent release w a bug fix?

### DISCUSSION THREAD

**Discussion by xke** [nithinmurali/pygsheets#532] [General]
Hi @nithinmurali ,

I've been using pygsheets and really like it!

Saw that you implemented a bug fix in June 2021 that fixes something I also ran into with duplicating sheets:
https://github.com/nithinmurali/pygsheets/issues/496

How do I install a version of pygsheets with that fix? It seems like the current most recent version 2.0.5 is from Feb 2021.

Thank you!

**Comment by nithinmurali** [comment]
Hi @xke , you can install from github `pip install https://github.com/nithinmurali/pygsheets/archive/staging.zip` .

I will make a new release sometime soon with the recent changes.

**Comment by xke** [comment]
Great!
@nithinmurali , i tried to install the staging file but it didn't seem like the changes from https://github.com/nithinmurali/pygsheets/issues/496 was reflected.

my use case is that I'm adding a duplicated worksheet to the beginning of the workbook. Will try the next release when you have it ready! Thanks again!
